opds.py

```diff
@@ -1538,6 +1513,16 @@ def acquisition_link(cls, rel, href, types):
             indirect_types = []
         link = cls.link(rel, href, initial_type)
         indirect = cls.indirect_acquisition(indirect_types)
+
+        if active_loan and \
+                active_loan.license_pool.collection.protocol == ExternalIntegration.LCP and \
+                active_loan.patron.external_identifier:
+            hashed_passphrase = AtomFeed.makeelement(
```

Comment:
Add a reference in a comment to
https://readium.org/lcp-specs/notes/lcp-key-retrieval.html#including-a-hashed-passphrase-in-an-opds-1-catalog
so it's clear this isn't something we made up.

Reply:
Added a comment